app/styles/style-flat.sass

```diff
@@ -564,6 +564,7 @@ body[lang='ru'], body[lang='uk'], body[lang='bg'], body[lang^='mk'], body[lang='
       letter-spacing: 0.48px
       line-height: 30px
       margin: 20px auto
+      display: block
```

Comment:
I think h3 should have display set to `block` by default on browsers. Interested why we're adding the explicit `display` and if this could cause style regressions?

Thanks! I like explicit styles so happy with this change, just clarifying what issue prompted this addition.

Reply:
https://github.com/codecombat/codecombat/blob/master/app/styles/courses/teacher-class-view.sass#L20-L21

It's overwrote in this page, so have to add the explicit style